torch_geometric/transforms/add_node_feature.py

```diff
@@ -0,0 +1,56 @@
+from typing import Tuple, Union
+
+import torch
+from torch import Tensor
+
+from torch_geometric.data import Data, HeteroData
+from torch_geometric.data.datapipes import functional_transform
+from torch_geometric.transforms import BaseTransform
+
+
+@functional_transform('add_node_feature')
+class AddNodeFeature(BaseTransform):
```

Comment:
`Add` is a bit confusing since we are concatenating features instead of summing them up.